conversational-client/src/app/app.component.html

```diff
@@ -3,7 +3,7 @@
     ><app-mode-selector></app-mode-selector
   ></mat-sidenav>
 
-  <mat-sidenav-content id="content">
+  <mat-sidenav-content id="content" #content>
```

Comment:
we need both because they serve slightly different purposes. `#content` is an angular template variable; we need it get the element reference to scroll the page. we need `id="content"` to style the element.